templates/docs/examples/patterns/navigation/_script-sliding.js

```diff
@@ -44,22 +60,38 @@ const initNavigationSliding = () => {
     }
   });
 
+  const secondaryNavToggle = document.querySelector('.js-secondary-menu-toggle-button');
+  secondaryNavToggle.addEventListener('click', (event) => {
+    event.preventDefault();
+    closeSearch();
+    if (secondaryNavigation.classList.contains('has-menu-open')) {
+      closeAll();
+    } else {
+      secondaryNavigation.classList.add('has-menu-open');
+      let icon = secondaryNavToggle.querySelector('.p-icon--chevron-down');
+      icon.classList.remove('p-icon--chevron-down');
+      icon.classList.add('p-icon--chevron-up');
+    }
+  });
+
   const resetToggles = (exception) => {
     toggles.forEach(function (toggle) {
       const target = document.getElementById(toggle.getAttribute('aria-controls'));
-      if (target === exception) {
+      if (!target || target === exception) {
         return;
       }
+      console.log(toggle, target);
```

Comment:
Nit: remove unnecessary logging statements

Reply:
It's not a nit, it's something that definitely should be cleared out ;)